source/_components/azure.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: page
+title: "Azure"
+description: "Instructions on how to setup the Azure component."
+date: 2018-08-28 10:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: microsoft_azure.png
+ha_category: Others
+featured: true
+ha_release: 0.77.1
```

Comment:
New platform and components are only added to major releases. Thus, `0.78` or `0.79`.